lib/logstash/outputs/pagerduty.rb

```diff
@@ -33,6 +33,11 @@ class LogStash::Outputs::PagerDuty < LogStash::Outputs::Base
   config :pdurl, :validate => :string, :default => "https://events.pagerduty.com/generic/2010-04-15/create_event.json"
 
   public
+  def set_client(client)
+    @client 
+  end
```

Comment:
This method accept an arguments but doesn't do anything with it?
Also I don't see mention of this method anywhere in this diff, are we really using it?